### ISSUE TITLE
Splade Confirm: Extraneous non-props attributes (default-password-text, confirm-password-route) were passed to component

### ISSUE THREAD

**Issue by JapSeyz** [protonemedia/laravel-splade#268]
- Laravel Version: 9.48.0
- PHP Version: 8.1.14
- Splade JS Version (npm): 1.2.4
- Splade PHP Version (composer): 1.2.4

You may use the [built-in Artisan Command](https://splade.dev/docs/upgrading) to show the installed versions of the packages:

```bash
php artisan splade:show-versions
```

### Description:

```javascript
[Vue warn]: Extraneous non-props attributes (default-password-text, confirm-password-route) were passed to component but could not be automatically inherited because component renders fragment or text root nodes. 
  at <SpladeConfirm default-title="Are you sure you want to continue?" default-text="" default-password-text="Please confirm your password before continuing"  ... > 
  at <Anonymous key=0 > 
  at <Render html="<SpladeConfirm\n    default-title=\"Are you sure you want to continue?\"\n    default-text=\"\"\n    default-password-text=\"Please confirm your password before continuing\"\n    default-confirm-button=\"Confirm\"\n    default-cancel-button=\"Annuller\"\n    confirm-password-route=\"\"\n>\n    <template #default=\"confirm\">\n        <SpladeTransition #default=\"{ TransitionRoot, TransitionChild }\">\n    <component\n        v-bind:unmount=\"true\" v-bind:appear=\"false\" v-bind:show=\"confirm.isOpen\"
  at <Anonymous key=0 > ....
```

From the console.
I am not using `confirm-text` and `confirm-button`, but never using the `default-password-text` etc. Therefore I think it's emitted from within the package.

**Comment by jamesj2** [comment]
Try updating the npm package @protonemedia/laravel-splade.

**Comment by iMusicJJ** [comment]
Yup, seems fixed in `1.2.5`. Sweet
